src/frontend/js/widgets/Dashboard/components/DashboardItem/Order/ContractFrame/index.tsx

```diff
@@ -0,0 +1,247 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Button, Loader } from '@openfun/cunningham-react';
+import { useQueryClient } from '@tanstack/react-query';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import { Modal } from 'components/Modal';
+import { useJoanieApi } from 'contexts/JoanieApiContext';
+import { Order, Product } from 'types/Joanie';
+import { SignatureDummy } from 'widgets/Dashboard/components/DashboardItem/Order/ContractFrame/SignatureDummy';
+import { SignatureLex } from 'widgets/Dashboard/components/DashboardItem/Order/ContractFrame/SignatureLex';
```

Comment:
In order to enlight JS bundle, what do you think to lazy load the right signature interface like we did with Payment interfaces ?